src/calendar/MonthView.java

```diff
@@ -0,0 +1,241 @@
+package calendar;
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.ArrayList;
+import java.awt.*;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import javax.swing.*;
+
+public class MonthView extends JPanel {
```

Comment:
MonthView also needs to extend ChangeListener, so that MonthView knows to update when the CalendarModel updates (part of MVC pattern)